info/v2/container.go

```diff
@@ -52,6 +52,14 @@ type MemorySpec struct {
 	SwapLimit uint64 `json:"swap_limit,omitempty"`
 }
 
+type ContainerInfo struct {
+	// Describes the container.
+	Spec ContainerSpec `json:"spec,omitempty"`
```

Comment:
#1035 doesn't change ContainerStats. This PR is changing the wire format for the `/stats` API right?